var/spack/repos/builtin/packages/r-a4/package.py

```diff
@@ -7,16 +7,21 @@
 
 
 class RA4(RPackage):
-    """Automated Affymetrix Array Analysis Umbrella Package."""
+    """Automated Affymetrix Array Analysis Umbrella Package.
 
-    homepage = "https://www.bioconductor.org/packages/a4/"
+       Automated Affymetrix Array Analysis Umbrella Package"""
```

Comment:
This description is duplicated. I assume this came from combining the short and long descriptions? I would either only use one of them, or check if the long description is the same (or shorter) than the short description.

Reply:
Updated to follow this criteria and it seems to have cleaned up most of the redundant descriptions.